app/controllers/api/v2/hosts_controller.rb

```diff
@@ -193,8 +193,10 @@ def get_status
         case params[:type]
           when 'global'
             @status = @host.build_global_status
-          else
+          when 'configuration', 'build'
```

Comment:
Note, that plugins may add other statuses here. These must also continue to work.
There is a status registry API to check for available statuses.
This needs test coverage.